account_invoice_production_lot/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module shows, for each (customer) invoice line, the delivered production
+lots that will be also displayed on the invoice report with their corresponding
+delivered quantities in case the **Tracking** has been set to **By Lots**.
```

Comment:
Delivered quantities always are shown, in both cases, when we tracking product by lots and by and by unique serial number.

Reply:
I have adapted the code to comply with the README.